### ISSUE TITLE
I can't select a node without .update()

### ISSUE THREAD

**Issue by lourenci** [airbnb/enzyme#1724]
Do you guys know why I can't find an element that is showing up in `console.log`?

I'm having to `update` the component to pass this test.

Test:
![image](https://user-images.githubusercontent.com/2339362/43235231-3cb2311e-9056-11e8-8d51-0a8d42cc13ae.png)

Result:
![image](https://user-images.githubusercontent.com/2339362/43235210-1e8d68e8-9056-11e8-8349-ccb88591dbbf.png)

**Comment by ljharb** [comment]
`.html()` calls `render` - what does `subject.debug()` output?

**Comment by lourenci** [comment]
@ljharb 
![image](https://user-images.githubusercontent.com/2339362/43258952-0e5b65f6-90ab-11e8-8ed9-fa66bc8b2ff9.png)

Now, let me call `update` before the `expect`:
![image](https://user-images.githubusercontent.com/2339362/43259112-87ea27b8-90ab-11e8-8ecb-d732a8077f57.png)

The result:
![image](https://user-images.githubusercontent.com/2339362/43259058-62590762-90ab-11e8-9a17-c896393f5448.png)

The spec does not fail.

**Comment by emuraton** [comment]
Does your `AjaxLoader` has some logic inside `componentDidMount`? 
It looks like your component cant work without this part.

If so its probably related to https://github.com/airbnb/enzyme/issues/1708

**Comment by lourenci** [comment]
@emuraton no, this component doesn't have the `componentDidMount` method.

**Comment by ljharb** [comment]
Yes, you still have to call `.update()` - `mount` doesn't mean that the wrapper updates itself automatically.
